ToDoApp/ToDoApp/ToDoListViewModel.swift

```diff
@@ -26,7 +26,7 @@ final class ToDoListViewModel: ObservableObject {
     }
     @Published private(set) var destination: Destination?
     @Published private(set) var state: State = .idle
-    private let taskService: ToDoService
+    @Published private(set) var taskService: ToDoService
```

Comment:
Why do we need task service to be a var, as well why do we need it to be published? View doesn't need to observe changes of reference to the Todo service

Reply:
it's fixed now, I just changed it to see whether it caused the bug I had, but the bug was fixed and not related to this.